Benchmark/cmake/modules/Benchmark.cmake

```diff
@@ -4,7 +4,7 @@ add_custom_target(run
 )
 
 # Utility function to automatically create the build target for the given check 
-macro(add_benchmark CHECKID)
+function(add_benchmark CHECKID)
```

Comment:
Any particular reason why?

Reply:
Or else I cannot `return()` from the function